documentation/modules/ROOT/pages/configuration/topic-auto-create-config.adoc

```diff
@@ -172,62 +324,134 @@ the group name. This will look like that in your connector JSON:
      ...
 }
 ----
+endif::community[]
+ifdef::product[]
+* To define a custom topic group, add a `topic.creation._<group_name>_.include` property to `spec.config` in the connector custom resource, followed
+by the configuration properties that you want to apply to topics in the custom group.
++
+The following example shows an excerpt of a custom resource that defines the custom topic creation groups `inventory` and `applicationlogs`:
++
+[source,yaml]
+----
+apiVersion: kafka.strimzi.io/v1beta1
+kind: KafkaConnector
+metadata:
+  name: inventory-connector
+...
+spec:
+...
+
+   config:
+... //<1>
+     topic.creation.inventory.include: "dbserver1\\.inventory\\.*"  //<2>
+     "topic.creation.inventory.replication.factor": 3
```

Comment:
remove the double quotes here

Reply:
Ah, thanks, I missed missed seeing the extra quotes. :+1: Fixed it.